### ISSUE TITLE
Broken ontology URL

### ISSUE THREAD

**Issue by lukas-koschmieder** [emmo-repo/EMMO#35]
The ontology URL (http://emmo.info/emmo.owl) (see [README.md](https://github.com/emmo-repo/EMMO#repository-description
)) is broken: HTTP 404 (Not Found).

**Comment by MarcZimmermann** [comment]
HTTP redirects to HTTPS (https://emmo.info/emmo.owl )

**Comment by lukas-koschmieder** [comment]
Are you saying that the HTTPS link works for you? For me, the error message is still the same.

**Comment by jesper-friis** [comment]
You should use http, which will be redirected to https. Both 
http://emmo.info/emmc and http://emmo.info/emmc/1.0.0-alpha works fine for me.

**Comment by lukas-koschmieder** [comment]
The HTTPS redirection works but I can't access the actual target file (HTTP 404). I've tested the above links on three different hosts/clients/networks: my office PC (Edge, ACCESS e.V.), a remote Linux VM (wget, RWTH Aachen University) and on my mobile phone (Chrome, Vodafone).

```
$ wget http://emmo.info/emmc/1.0.0-alpha
--2020-03-11 11:31:02--  http://emmo.info/emmc/1.0.0-alpha
Resolving emmo.info (emmo.info)... 185.61.153.115
Connecting to emmo.info (emmo.info)|185.61.153.115|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://emmo.info/emmc/1.0.0-alpha [following]
--2020-03-11 11:31:02--  https://emmo.info/emmc/1.0.0-alpha
Connecting to emmo.info (emmo.info)|185.61.153.115|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2020-03-11 11:31:02 ERROR 404: Not Found.
```

**Comment by jesper-friis** [comment]
You are right. wget doesn't work for me either. But it works fine to open the url in Protege and EMMO-python (Owlready2).

**Comment by lukas-koschmieder** [comment]
Protege 5.5.0 doesn't work for me either. Same error. But at least this makes sense in the way that all clients fail. What's weird is that in your case certain clients work (Protege) while others (wget)... Maybe there is something wrong with the certificate. I've noticed that it has be updated a month ago which is roughly the time it stopped working for me. This is just a shoot in the dark... did you guys maybe configure your Protege clients to always trust this server? This would also explain why wget doesn't work.

```
Full Stack Trace
-----------------------------------------------------------------------------------------

java.io.FileNotFoundException: https://emmo.info/emmo.owl
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:1926)
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:1921)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:1920)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1490)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1474)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:254)
	at org.semanticweb.owlapi.io.AbstractOWLParser.getInputStreamFromContentEncoding(AbstractOWLParser.java:179)
	at org.semanticweb.owlapi.io.AbstractOWLParser.getInputStream(AbstractOWLParser.java:141)
	at org.semanticweb.owlapi.io.AbstractOWLParser.getInputSource(AbstractOWLParser.java:264)
	at org.semanticweb.owlapi.rdf.rdfxml.parser.RDFXMLParser.parse(RDFXMLParser.java:72)
	at uk.ac.manchester.cs.owl.owlapi.OWLOntologyFactoryImpl.loadOWLOntology(OWLOntologyFactoryImpl.java:221)
	at uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.actualParse(OWLOntologyManagerImpl.java:1262)
	at uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.loadOntology(OWLOntologyManagerImpl.java:1215)
	at uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.loadOntologyFromOntologyDocument(OWLOntologyManagerImpl.java:1172)
	at org.protege.editor.owl.model.io.OntologyLoader.loadOntologyInternal(OntologyLoader.java:104)
	at org.protege.editor.owl.model.io.OntologyLoader.lambda$loadOntologyInOtherThread$0(OntologyLoader.java:62)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.FileNotFoundException: https://emmo.info/emmo.owl
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1872)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1474)
	at sun.net.www.protocol.http.HttpURLConnection.getHeaderField(HttpURLConnection.java:2982)
	at java.net.URLConnection.getContentEncoding(URLConnection.java:523)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getContentEncoding(HttpsURLConnectionImpl.java:410)
	at org.semanticweb.owlapi.io.AbstractOWLParser.getInputStream(AbstractOWLParser.java:136)
	... 12 more
```

**Comment by jesper-friis** [comment]
Sorry Lukas. It was me how spelled the URLs wrong in my reply. Both of the following do actually work:

    wget http://emmo.info/emmo
    wget http://emmo.info/emmo/1.0.0-alpha

**Comment by lukas-koschmieder** [comment]
Thank you, Jesper. These links also work for me. I've checked the README and Emanuele has fix the links in the meantime 174e282cf6ad255669b5108cfb312a05af3260dd.
